packages/frontend/src/pages/SavedDashboards.tsx

```diff
@@ -52,6 +52,7 @@ const SavedDashboards = () => {
                     const { uuid } = savedDashboard;
                     return `/projects/${projectUuid}/dashboards/${uuid}/view`;
                 }}
+                // @ts-ignore
```

Comment:
why do we need this ignore here ?

Reply:
https://github.com/lightdash/lightdash/pull/1707#discussion_r849244691